.eslintrc.json

```
@@ -10,6 +10,7 @@
         "@stylistic"
     ],
     "rules": {
+        "unicorn/no-typeof-undefined": "off",
```

Biome (1.9.4)

[error] The key unicorn/no-typeof-undefined was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)